hca-dcp/submit.wdl

```diff
@@ -0,0 +1,163 @@
+# Get Cromwell metadata for the workflow that produced the given output
+task get_metadata {
+  String analysis_output_path
+
+  command <<<
+
+    # Get workflow_id
+    python <<CODE > workflow_id.txt
+
+    # Extract hash given a string like gs://foo/hash/call-bar/baz.txt
+    url = '${analysis_output_path}'
+    hash_end = url.rfind("/call-")
```

Comment:
I was going to suggest a regular expression here, but then couldn't be bothered to write it myself, so that's probably a sign that this is good enough 😬